### PR TITLE
Add use of service in dynamic validation group

### PR DESCRIPTION
**Pull request by pierre-H**
Add doc for https://github.com/api-platform/core/pull/1198

### CODE REVIEW

**Comment by dunglas** [comment]
I made some slight changes. Can you double check?

File: core/validation.md [hunk 1]
```diff
@@ -95,6 +95,70 @@ class Book
...
+
+```php
+<?php
+
+// src/AppBundle/Entity/Book.php
+
+namespace AppBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use AppBundle\Validator\AdminGroupsGeneratorGroupsGenerator;
```
Comment (soyuka) [inline core/validation.md]: Hmm is it not `+use AppBundle\Validator\AdminGroupsGenerator`?

Comment (Simperfit) [reply core/validation.md]: You are right.

**Comment by meyerbaptiste** [comment]
Thanks @pierre-H!

**Pull request merged**